src/auth/auth-router.js

```diff
@@ -0,0 +1,45 @@
+const express = require('express');
+const authRouter = express.Router();
+
+const authManager = require('./auth-manager');
+
+// Get the current User's role.
+authRouter.get('/', authManager.verify, (req, res) => {
+  if (!res.locals.auth) {
+    res.status(400).send('Auth failed.').end();
```

Comment:
What about the errors that would have been passed up to the stack to this point?